python/py-circuitbreaker/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-circuitbreaker
+version             2.0.0
+
+license             BSD-3-Clause
+platforms           {darwin any}
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         Python implementation of the \"Circuit Breaker\" Pattern
+long_description    {*}${description}
+
+homepage            https://github.com/fabfuel/circuitbreaker
+
+checksums           md5     0fc04e58267a46ac371aac0f3e2b0105 \
+                    rmd160  992822cb56baf69b33a3da002c9ebbf5dfa56ddd \
+                    sha256  28110761ca81a2accbd6b33186bc8c433e69b0933d85e89f280028dbb8c1dd14 \
+                    size    10736
+
+python.rootname     circuitbreaker
+python.versions     37 38 39 310 311
+
+if {${subport} ne ${name}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_test-append \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-pytest-asyncio \
+                    port:py${python.version}-pytest-mock \
+                    port:py${python.version}-pytest-cov \
+                    port:py${python.version}-coverage \
+                    port:py${python.version}-flake8
```

Comment:
the `python` PG sets things automatically (see comment below). For example, the `pytest` dependency is added automatically and can thus be removed here. Additionally, we don't typically test coverage within MacPorts (and remove this by setting `test.args -o addopts=` as default. So I *think* you can safely remove the `pytest-cov`, `coverage` and `flake8` dependencies. Just make sure to re-run the test-phase to verify that the test indeed still pass.